packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/document/DocumentAsset.java

```diff
@@ -60,6 +60,10 @@ public class DocumentAsset extends TerariumAsset {
 	@JoinColumn(name = "grounding_id")
 	private Grounding grounding;
 
+	@TSOptional
+	@Column(columnDefinition = "text")
+	private String documentAbstract;
```

Comment:
```suggestion
	private String abstract;
```

Reply:
This suggestion won't work - `abstract` is a reserved keyword in Java